test/api/v3/unit/middlewares/getUserLanguage.test.js

```diff
@@ -12,6 +13,11 @@ import accepts from 'accepts';
 describe('getUserLanguage', () => {
   let res, req, next;
 
+  let checkReqT = (req) => {
+    expect(res.t).to.be.a('function');
+    expect(res.t('help')).to.equal(i18n.t('help', req.language));
```

Comment:
Can you explain it better, I'm not getting it. (yeah, the name should be checkResT)